run.py

```diff
@@ -184,7 +184,9 @@ def process(config: PathLike,
         script_paths = [*SCRIPT_PATHS['init'], *script_paths]
 
     for script_path in script_paths:
-        step_name = script_path.name.replace('.py', '')[3:]
```

Comment:
Changed because it did not work with file names such as `recon_all.py` or `04a-run_ica.py`